Example/TABTestKit_ExampleUITests/OtherElementsTests.swift

```diff
@@ -166,6 +167,94 @@ final class OtherElementsTests: TABTestCase, SystemPreferencesContext {
             When(I: adjust(otherElementsScreen.picker.wheel(0), to: "World"))
             Then(the: value(of: otherElementsScreen.picker.wheel(0), is: "World"))
         }
+        
+        Scenario("Seeing and interacting with the time picker") {
```

Comment:
Couple tests with differents variations of all date pickers (Date, Time, DateTime and Count Down Time)